drivers/net/mlx5/mlx5_vdpa.c

```diff
@@ -39,8 +43,48 @@ static struct vdpa_priv_list_head priv_list =
 					TAILQ_HEAD_INITIALIZER(priv_list);
 static pthread_mutex_t priv_list_lock = PTHREAD_MUTEX_INITIALIZER;
 
+static struct vdpa_priv_list *
+find_priv_resource_by_did(int did)
```

Comment:
I knida copied that from ifcvf, and I think it's more natural to return a linked-list node in a search function, rather then the inside, since the caller can traverse it from there (it it wants too).

Reply:
OK then.